pkg/scd/dss_report_handler.go

```diff
@@ -0,0 +1,36 @@
+package scd
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/google/uuid"
+	restapi "github.com/interuss/dss/pkg/api/scdv1"
+	"github.com/interuss/dss/pkg/logging"
+	"github.com/interuss/stacktrace"
+	"go.uber.org/zap"
+)
+
+// JSONLoggingDssReportHandler a DssReportHandler that simply logs the received report as JSON.
+type JSONLoggingDssReportHandler struct {
```

Comment:
This is not the convention used in the other handlers. I'd suggest removing this struct and just embedding the code in `MakeDssReport` given it is that simple. We are just using the normal logger anyway, right?

Reply:
Fair :+1: